pkg/kubelet/apis/config/types.go

```diff
@@ -679,3 +685,11 @@ type MemorySwapConfiguration struct {
 	// +optional
 	SwapBehavior string
 }
+
+type CrashLoopBackOffConfig struct {
+	// maxSeconds is the maximum number of seconds the backoff delay can accrue
+	// to for container restarts
```

Comment:
nit: document the minimum & maximum constraints here

Reply:
and default, if we go that way

Reply:
Fixed in 6692dda, TBD default